post-office-stub/src/MockPostOfficeHandler.ts

```diff
@@ -0,0 +1,59 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { Metrics } from "@aws-lambda-powertools/metrics";
+import { Response } from "./utils/Response";
+import { HttpCodesEnum } from "./utils/HttpCodesEnum";
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+import { AppError } from "./utils/AppError";
+import { PostOfficeRequestProcessor } from "./services/PostOfficeRequestProcessor";
+
+
+const POWERTOOLS_METRICS_NAMESPACE = process.env.POWERTOOLS_METRICS_NAMESPACE ? process.env.POWERTOOLS_METRICS_NAMESPACE : Constants.F2F_METRICS_NAMESPACE;
+const POWERTOOLS_LOG_LEVEL = process.env.POWERTOOLS_LOG_LEVEL ? process.env.POWERTOOLS_LOG_LEVEL : Constants.DEBUG;
+const POWERTOOLS_SERVICE_NAME = process.env.POWERTOOLS_SERVICE_NAME ? process.env.POWERTOOLS_SERVICE_NAME : Constants.AUTHORIZATIONCODE_LOGGER_SVC_NAME;
```

Comment:
AUTHORIZATIONCODE_LOGGER_SVC_NAME? Think we need to create a new Powertools service name here for this lambda